### ISSUE TITLE
Trimmed filenames on  check

### ISSUE THREAD

**Issue by dd86k** [dd86k/ddh#1]
```
error: 'mobian-installer-pinephone-phosh-20210516.img.g': Cannot open file `mobian-installer-pinephone-phosh-20210516.img.g' in mode `rb' (No such file or directory)
```

**Comment by dd86k** [comment]
In binary mode, `byLine` only, at best, processes `\n`.
In text mode, `byLine` processes `\n` and `\r\n` (on Windows, at least).

Writing something quick.

**Comment by dd86k** [comment]
Fixed in 3acad88e4b2610e9d045bd9dfc31ee652047d1e4 so keeping this open until closed
